### PR TITLE
fix(FEC-9262): playback doesn't return to start after playback with ads in LG TV

### PR DESCRIPTION
**Pull request by Yuvalke**
### Description of the Changes

Add support for replay after ads finish to return to beginning. 
fix bug engine.ended on `attachMediaSource` will not give always true on the end of the media as expected, instead on `detachMediaSource` which will return true when ad playing and decorator is active or when decorator is not active engine will return true from video element.
keeps the value form `detachMediaSource` to know in `attachMediaSource` if media ended.

### CheckLists

- [ ] changes have been done against master branch, and PR does not conflict
- [ ] new unit / functional tests have been added (whenever applicable)
- [ ] test are passing in local environment
- [ ] Travis tests are passing (or test results are not worse than on master branch :))
- [ ] Docs have been updated

### CODE REVIEW

File: src/engines/html5/html5.js [hunk 1]
```diff
@@ -253,17 +253,7 @@ export default class Html5 extends FakeEventTarget implements IEngine {
    */
   attachMediaSource(): void {
     if (this._mediaSourceAdapter) {
-      //added to mask problematic behavior in shaka - init fire loadstart event, only for last init to avoid spinner
       this._mediaSourceAdapter.attachMediaSource(this._playbackEndedOnDetach);
-      if (this._playbackEndedOnDetach) {
```
Comment (OrenMe) [inline src/engines/html5/html5.js]: @Yuvalke so what is the solution here if you remove this?

Comment (Yuvalke) [reply src/engines/html5/html5.js]: Shaka init on load, so i didn't init on attaching cause when load call it'll be init again.
On the end load doesn't happen

Comment (OrenMe) [reply src/engines/html5/html5.js]: Good news, so we can wrap this with the other PRs or anything else is left?

Comment (Yuvalke) [reply src/engines/html5/html5.js]: I have a small fix which will remove the parameters, there are not needed anymore.
Every adapter can handle every by himeself

Comment (Yuvalke) [reply src/engines/html5/html5.js]: I have a small fix which will remove the parameters, there are not needed anymore.
Every adapter can handle every by himeself

**Pull request merged**